src/site/stages/build/drupal/metalsmith-drupal.js

```diff
@@ -59,6 +59,7 @@ function pipeDrupalPagesIntoMetalsmith(contentData, files) {
     const pageCompiled = compilePage(page, contentData);
     const drupalPageDir = path.join('.', drupalUrl);
     const drupalFileName = path.join(drupalPageDir, 'index.html');
+    const layoutName = entityBundle || 'page';
```

Comment:
We had a few failures when the entityBundle was undefined, it couldn't pick a template. This just sets "page" as the default.